port_ocean/tests/core/handlers/entity_processor/test_jq_entity_processor.py

```diff
@@ -269,3 +269,35 @@ async def test_parse_items_performance_10000(
             assert len(result.entity_selector_diff.passed) == 1
             assert result.entity_selector_diff.passed[0].properties.get("foo") == "bar"
             assert not result.errors
+
+    async def test_parse_items_wrong_mapping(
```

Comment:
Let's also add tests for different kinds of mappings which are supposed to be valid (i.e. items to parse, file kinds etc..)